trestle/core/markdown/base_markdown_node.py

```diff
@@ -148,14 +148,8 @@ def delete_nodes_text(self, keys: List[str], strict_matching: bool = True) -> Li
             delete_list_from_list(text_lines, list(range(node.starting_line, last_line)))
         return text_lines
 
-    def _build_tree(
-        self,
-        lines: List[str],
-        root_key: str,
-        starting_line: int,
-        level: int,
-        governed_header: Optional[str] = None
-    ) -> Tuple[BaseMarkdownNode, int]:
+    def _build_tree(self, lines: List[str], root_key: str, starting_line: int,
+                    level: int) -> Tuple[BaseMarkdownNode, int]:
```

Comment:
I think this function is dead and is not being called.  `level` isn't used and the function has no code coverage.  But it still may be needed in terms of the overall API...

Reply:
Good catch, I meant to replace this with `pass` as child classes should implement it